src/CMakeLists.txt

```diff
@@ -118,6 +119,7 @@ ADD_EXECUTABLE(lmms
 	core/main.cpp
 	$<TARGET_OBJECTS:lmmsobjs>
 	"${WINRC}"
+	"sys/Time.cpp"
```

Comment:
You generally shouldn't need to add files to the `lmms` target - `lmmsobjs` should be enough. We also prefer to add files to the `LMMS_SRCS` or `LMMS_INCLUDES` variables instead of `lmmsobjs` directly.

Reply:
fixed